build-support/bin/act/BUILD

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# This contains all the plumbing in order to get https://github.com/nektos/act working in a sandbox
+# and in `run`.
+# To run `act`, use `pants run build-support/bin/act -- --version` (or whatever args you want).
+
+file(
+    name="downloaded-act",
+    source=per_platform(
```

Comment:
This right here is the business!

Reply:
♥️

I tried doing something similar with GH, but it was sometimes a tar, sometimes not, and just didn't seem worth it at the time.

Maybe I'll try again as well ..